.github/CODEOWNERS

```diff
@@ -48,23 +48,23 @@
 
 
 # Angular Migration team
-/core/templates/components/shared-component.module.ts @srijanreddy98
-/core/templates/services/loader.service*.ts @srijanreddy98
-/core/templates/components/material.module.ts @srijanreddy98
-/core/templates/components/oppia-angular-root.component.ts @srijanreddy98
-/core/templates/base-components/oppia-root.directive.ts @srijanreddy98
-/core/templates/base-components/oppia-root.directive.html @srijanreddy98
-/core/templates/filters/translate.pipe.ts @srijanreddy98
-/core/templates/services/angular-services.index.ts @srijanreddy98
-/core/templates/services/contextual/logger.service.ts @srijanreddy98
-/core/templates/services/contextual/logger.service.spec.ts @srijanreddy98
-/core/templates/services/i18n-language-code.service.ts @srijanreddy98
-/core/templates/services/loader.service*.ts @srijanreddy98
-/core/templates/services/question-validation.service*.ts @srijanreddy98
-/core/templates/services/request-interceptor.service*.ts @srijanreddy98
-/core/templates/services/translate.service.ts @srijanreddy98
-/core/templates/services/translations-backend-api.service.ts @srijanreddy98
-/core/templates/services/UpgradedServices.ts @srijanreddy98
+/core/templates/components/shared-component.module.ts @srijanreddy98 @darksun27
```

Comment:
@srijanreddy98 is this a temporary change?

Reply:
No